doc/user/isisd.rst

```diff
@@ -497,12 +506,12 @@ which flex algorithm they must use for a given packet.
 The following commands configure Flex-Algo at the 'router isis' configuration
 level. Segment-Routing prefixes must be configured for the Flex-Algo.
 
-.. clicmd:: flexible-algorithm (128-255)
+.. clicmd:: flex-algo (128-255)
 
    Add a Flex-Algo Definition (FAD) and enter the FAD configuration
    level. The algorithm ID value is in the range of 128 to 255 inclusive.
 
-.. clicmd:: no flexible-algorithm (128-255)
+.. clicmd:: no flex-algo (128-255)
```

Comment:
This can be removed at all. We do not store `no` forms at all basically.

Reply:
Done, thanks.